scm-ui/ui-webapp/src/containers/NavigationBar.tsx

```diff
@@ -93,14 +112,9 @@ type Props = {
   links: Links;
 };
 
-const BurgerActionBar: FC = () => (
-  <div className="navbar-burger-actions">
-    <Notifications className="navbar-item" direction="left" />
-  </div>
-);
-
 const NavigationBar: FC<Props> = ({ links }) => {
   const [burgerActive, setBurgerActive] = useState(false);
+  const [matchQuery, setMatchQuery] = useState<boolean>(false);
```

Comment:
```suggestion
  const [matchQuery, setMatchQuery] = useState(false);
```

The boolean generic is not necessary TypeScript is able to infer the type from the initial value.